sdcm/monitorstack/__init__.py

```diff
@@ -299,17 +329,13 @@ def run_monitoring_stack_containers(monitoring_stack_dir, monitoring_data_dir, s
 
 
 @retrying(n=3, sleep_time=20, message='Uploading sct dashboard')
-def restore_sct_dashboards(monitoring_dockers_dir, grafana_docker_port):
-    sct_dashboard_file_name = "*scylla-dash-per-server-nemesis.*.json"
-    sct_dashboard_files = list((Path(monitoring_dockers_dir) / 'sct_monitoring_addons').glob(sct_dashboard_file_name))
-
-    if not sct_dashboard_files or not os.path.exists(sct_dashboard_files[0]):
-        LOGGER.warning('There is no dashboard %s. defaults to master dashboard', sct_dashboard_file_name)
+def restore_sct_dashboards(grafana_docker_port, sct_dashboard_file):
+    if not sct_dashboard_file:
+        LOGGER.warning('There is no dashboard %s. defaults to master dashboard', sct_dashboard_file)
         sct_dashboard_file_name = "scylla-dash-per-server-nemesis.master.json"
-        sct_dashboard_files = [Path(__file__).parent.parent.parent / 'data_dir' / sct_dashboard_file_name]
+        sct_dashboard_file = [Path(__file__).parent.parent.parent / 'data_dir' / sct_dashboard_file_name]
 
     dashboard_url = f'http://localhost:{grafana_docker_port}/api/dashboards/db'
-    sct_dashboard_file = sct_dashboard_files[0]
```

Comment:
I think removal of that line must break the code. On the following line with `with` statement we expect there a string-like object not list.

Have you tested it?

Reply:
to get to a point you don't have a dashboard, is was back when we used the per version dashboard, and it was missing.
it's rarely would happen now.

but yes this fallback won't work...